src/styles/blocks/header.scss

```diff
@@ -0,0 +1,67 @@
+.header {
+  display: flex;
+  justify-content: space-between;
+  align-items: center;
+  padding: 0 50px;
+
+  &__logo {
+    display: block;
+  }
+}
+
+.nav {
+  display: flex;
+
+  &__list {
+    margin: 0;
+    padding: 0;
+
+    display: flex;
+    align-items: center;
+    list-style-type: none;
+  }
+
+  &__item {
+    display: flex;
+    align-items: center;
+    position: relative;
+
+    &:not(:last-child) {
+      margin-right: 20px;
+    }
+  }
+
+  &__link {
+    font-family: "Roboto", Arial, sans-serif;
+    text-decoration: none;
+    color: $color-black;
+    text-transform: uppercase;
+    font-size: 12px;
+    font-weight: 500;
+    line-height: 14px;
+    letter-spacing: 0;
+    text-align: left;
+    transition: all 0.3s;
```

Comment:
don't use it like that, because it's the same as transition-duration: 0.3s
It's too hard for the browser
Check this
![image](https://user-images.githubusercontent.com/91411692/222670572-70278fda-3d5f-4b0e-950b-56a04d76d44c.png)

```suggestion
    transition: color $transitionDuration;
```